nativeApp/src/screens/admin_views/Volunteers/EditVolunteer/index.tsx

```diff
@@ -18,10 +21,24 @@ type Props = {
 /*
  * Component
  */
-const EditVolunteer: FC<NavigationInjectedProps & Props> = (props) => (
-  <Page heading="Edit Volunteer">
-    <UserForm onSubmit={() => props.navigation.navigate('Volunteers')} />
-  </Page>
-);
+
+// TODO add types
+const EditVolunteer: FC<NavigationInjectedProps<any> & Props> = ({ navigation }) => {
+  const onSubmit = async (formState) => {
+    try {
+      const changeset = filter(Boolean, formState);
+      await CommunityBusinesses.editVolunteer(changeset); // TODO not use ramda
```

Comment:
Why `TODO not use ramda`?

Reply:
See previous comment

Reply:
Was just reminded that the typings on this ramda function meant that it assumed changeset was an array rather than an object, and was breaking types here. This is what prompted my comment